tools/releaseBuild/azureDevOps/templates/nuget.yml

```diff
@@ -201,16 +201,18 @@ jobs:
       $packagePathList = Get-ChildItem $packagePath -Recurse | Select-Object -ExpandProperty FullName | Out-String
       Write-Verbose -Verbose $packagePathList
 
-      Get-ChildItem -Path $packagePath | 
-          ForEach-Object {
-            Write-Verbose -Verbose "Generating checksum file for $($_.FullName)"
-            $packageName = $_.Name
-            $hash = (Get-FileHash -Path $_.FullName -Algorithm SHA512).Hash.ToLower()
-
-            # the '*' before the packagename signifies it is a binary
-            "$hash *$packageName"
-          } |
-          Out-File -FilePath "$packagePath\SHA512SUMS" -Force
+      $checksums = Get-ChildItem -Path $packagePath |
```

Comment:
The only thing changed here is assigning to a variable before piping. Does that have any effect? Do we want to log the value or something?

Reply:
We write the file to the same path Get-ChildItem is looking at. That causes "file is in use". So writing the file after all hashes are computed.